test/support/test.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
@norcal82 no, im using it here https://github.com/amco/dolly/pull/94/files#diff-ac81ef2b8c0a77ed03ef17846623b563R453